include/cufinufft_opts.h

```diff
@@ -20,6 +20,9 @@ typedef struct cufinufft_opts {   // see cufinufft_default_opts() for defaults
 	int gpu_kerevalmeth; // 0: direct exp(sqrt()), 1: Horner ppval
 
 	int gpu_spreadinterponly; // 0: NUFFT, 1: spread or interpolation only
+
+        /* multi-gpu support */
+        int cuda_device_id;
```

Comment:
What about just `device_id`? It's going to be CUDA in any case, right?

Reply:
sure!